src/cellfinder_core/tools/array_operations.py

```diff
@@ -4,7 +4,7 @@
 
 
 # WARNING: skewed if does not fall exactly (integer multiple)
```

Comment:
should we add a docstring to `bin_array()` and mention this there instead?

Reply:
I've added notes sections to `bin_array()`, `pad_and_bin_mean()`, and `bin_mean_3d()` to clarify this (and removed the warning comment)